.github/workflows/publish-release.yml

```diff
@@ -50,7 +67,7 @@ jobs:
           fail-on-cache-miss: true
       - name: Dry Run Publish
         # omit npm-token token to perform dry run publish
-        uses: MetaMask/action-npm-publish@v4
+        uses: MetaMask/action-npm-publish@v5
```

Comment:
Nit: Was there a reason for updating to v5 in this PR? Seems like a good idea but it wasn't obvious how that related to the Yarn v4 update.

Reply:
Ah, good question, I didn't highlight this. Yarn v4 now requires passing an `--all` option to `yarn workspaces foreach` if you want to iterate over all packages in the workspace. `action-npm-publish` v4 didn't pass this option, but v5 does now: https://github.com/MetaMask/action-npm-publish/commit/f861a2bcfe16e6678e96161dbfeb37ca907c23da

Reply:
Ah, makes sense!